src/components/modal/mine-filter/mine-filter-modal.tsx

```diff
@@ -36,7 +36,7 @@ const HiddenIfOppdaterFilter = hiddenIf(OppdaterMineFilter);
 const HiddenIfFnrFeil = hiddenIf(MineFilterFnrFeil)
 
 export function MineFilterModal(props: { filtergruppe: string }) {
-    const {sisteValgteMineFilter, valgtMineFilter, erModalApen} = useSelector((state: AppState) => (props.filtergruppe === ListevisningType.minOversikt) ? state.mineFilterMinOversikt : state.mineFilterEnhetensOversikt)
+    const {sisteValgtLagretFilter, valgtLagretFilter, erModalApen} = useSelector((state: AppState) => (props.filtergruppe === ListevisningType.minOversikt) ? state.mineFilterMinOversikt : state.mineFilterEnhetensOversikt)
```

Comment:
aren't these also mine filter?

Reply:
yep

Reply:
This one is bit tricky, since this can also be used in veileder oversikt...

Reply:
mine filter is not in veilederoversikt

Reply:
yep, the trouble is with state.mineFilterMinOversikt (as you can see in reducers, all 3 variables (mineFilterMinOversikt, mineFilterEnhetensOversikt, mineFilterVeilederOversikt) point to the same reducer LagretFilterUIState (https://github.com/navikt/veilarbportefoljeflatefs/blob/e7e5ca971f303ed3fbb74662b030c931cb71bae1/src/ducks/lagret-filter-ui-state.ts#L17)...so thats why I use _sisteValgtLagretFilter_, _valgtLagretFilter_

Reply:
But this is inside the modal for MineFilter, so it does not make any sense to me that it's lagret filter -> both mine filter and veiledergrupper?

Reply:
Sorry, this is fixed by last refactoring, when we changes attributes in _LagretFilterUIState_...so it should be good now I think.

Reply:
Not sure if I understand what you mean.. it's still `sisteValgtLagretFilter`

Reply:
git is showing outdated file, pls check: https://github.com/navikt/veilarbportefoljeflatefs/blob/RefactoringAndBugfix/src/components/modal/mine-filter/mine-filter-modal.tsx

Reply:
Aaaah, sorry! Looks good now :D